src/main/java/seedu/address/model/person/Group.java

```diff
@@ -30,6 +30,10 @@ public Group(String group) {
         this.value = group;
     }
 
+    public Group() {
+        this.value = "";
```

Comment:
Perhaps a better way to do this is to have a cached Group object that represents an empty/missing group? Then, we can call a static method to retrieve the cached object when necessary.